ui/app/rif/components/lumino/open-channel/index.js

```diff
@@ -115,8 +124,8 @@ class OpenChannel extends Component {
       )
     }
 
-    if (!this.state.selectedToken) {
-      return (<div>Loading...</div>)
+    if (!this.state.selectedToken || !tokensOptions || (tokensOptions && tokensOptions.length <= 0)) {
```

Comment:
I find this really hard to understand without any previous idea of the code. Can you please add a comment that represent the condition?

Reply:
Basically it's a control since the component can have tokenOptions undefined or empty, that's because the service that retrieves the tokens can delay a little some times so we show the loader when that happens, i can add a comment although.